arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx

```diff
@@ -199,7 +198,7 @@ const CalculateRiskMeasurement: React.FC<IProps> = ({
           40
         )
         placeholders.text(`Ballot Number: ${ballot.position}`, 20, 100)
-        pageCount++
+        pageCount += 1
```

Comment:
Does the new eslint version now allow `++`? Or is this just a stylistic preference?

Reply:
The newer ESLint includes [`no-plusplus`](https://eslint.org/docs/rules/no-plusplus) by default.